### PR TITLE
update package

### PR DESCRIPTION
**Pull request by lucaferranti**

### CODE REVIEW

**Comment by lucaferranti** [comment]
@dpsanders this should be ready now 🎉 the workflow checks aren't showed here because there's no `.github/workflows` folder in the master branch (I think), but you can check the CI results on my fork [here](https://github.com/lucaferranti/ValidatedNumerics.jl/actions). The checks should then be triggered once this is merged.

File: Project.toml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+[compat]
+IntervalArithmetic = "0.16, 0.17, 0.18"
+IntervalRootFinding = "0.5"
+IntervalContractors = "0.4"
+IntervalConstraintProgramming = "0.12"
+IntervalOptimisation = "0.4"
+TaylorModels = "0.3"
+Reexport = "0.2, 1.0"
+julia = "1.3, 1.4, 1.5"
```
Comment (dpsanders) [inline Project.toml:23]: Add 1.6 presumably?

**Comment by dpsanders** [comment]
Great, thanks!

File: Project.toml (hunk 1 above)
Comment (lucaferranti) [reply Project.toml]: done!

**Comment by dpsanders** [comment]
Since we're updating the Julia version etc. this should be a new minor version.

**Comment by dpsanders** [comment]
Thanks!

**Pull request merged**